packages/react-server/core/util/LABString.js

```diff
@@ -1,11 +1,26 @@
-// Careful: We've got a local patch here!
-//
-// We've added support for a `crossOrigin` parameter.
-//
+const path = require("path");
+const {readFileSync} = require("fs");
+
+var lab = readFileSync(path.join(__dirname, "LAB.src.js"), "utf-8");
+
+// Don't need this.
+lab = lab.replace(/^"use strict";/, "");
+
+// Run with `DEBUG_LAB=true` to see its output in console.log.
+if (!process.env.DEBUG_LAB) { // eslint-disable-line
+
+	// This is exactly what @getify does to remove debug code.
+	// https://github.com/getify/LABjs/blob/master/BUILD.md
+	lab = lab.replace(/\/\*!START_DEBUG(?:.|[\n\r])*?END_DEBUG\*\//g, "");
+
+	// After the debug code is removed just uglify.
+	lab = require("uglify-js").minify(lab, {fromString: true}).code;
```

Comment:
> What am I missing?

The full extent of my laziness. 👹 

As I think about it more I like @doug-wade's idea of pulling this out into a separate package.  Going to manage it inside the React Server repo for now just for convenience.  We'll be able to pull it out later using [`lerna export`](https://github.com/lerna/lerna/pull/486) if we want to.